examples/imdb_fasttext.py

```diff
@@ -119,12 +119,9 @@ def add_ngram(sequences, token_indice, ngram_range=2):
                     embedding_dims,
                     input_length=maxlen))
 
-# we add a AveragePooling1D, which will average the embeddings
+# we add a Lambda layer, which will average the embeddings
 # of all words in the document
-model.add(AveragePooling1D(pool_length=model.output_shape[1]))
-
-# We flatten the output of the AveragePooling1D layer
-model.add(Flatten())
+model.add(Lambda(lambda x: K.mean(x, axis=1), output_shape=lambda s:(s[0], s[2])))
```

Comment:
Right. These layers currently don't support masking, but it's something we should add for sure.